src/dlgColorTrigger.cpp

```diff
@@ -33,10 +33,7 @@
 #include "post_guard.h"
 
 
-dlgColorTrigger::dlgColorTrigger( QWidget * pF, TTrigger * pT, int mode )
-: QDialog( pF )
-, mpTrigger( pT )
-, mMode( mode )
+dlgColorTrigger::dlgColorTrigger(QWidget* pF, TTrigger* pT, int mode) : QDialog(pF), mpTrigger(pT), mMode(mode)
```

Comment:
Sorry to pick at something that must be getting a bit scabbed by now but didn't we find a way/want to have the initialisations one per line?